modules/viewer/src/nextjournal/viewer/markdown.cljs

```diff
@@ -34,15 +35,26 @@
          :todo-item (fn [ctx {:as node :keys [attrs]}]
                       (md.transform/into-markup [:li [:input {:type "checkbox" :default-checked (:checked attrs)}]] ctx node))
                                                                                ;; ⬆ defaultChecked only makes senst in react/reagent
-         :formula (fn [_ctx node] [inspect* {:nextjournal/viewer :latex
```

Comment:
An `inspect` here would always wrap the latex markup string in a `[:div ` which would always render as block.